### ISSUE TITLE
No syntax highlighting for fresh install in Jupyter Notebook

### ISSUE THREAD

**Issue by awa5114** [dotnet/interactive#2827]
### Describe the bug
I installed the .NET interactive kernel as described by [this page.](https://github.com/dotnet/interactive/blob/main/docs/NotebookswithJupyter.md). The installation was successful and I can run .NET code no problem, but it looks like the syntax highlighting is missing:

### Please complete the following:

**Which version of .NET Interactive are you using?** (In a notebook, run the `#!about` magic command. ):
![image](https://user-images.githubusercontent.com/24267968/224748564-6ea5d568-7474-42ef-97fc-1bfbb713fa22.png)

- OS
    - [ ] Windows 11
    - [x] Windows 10
    - [ ] macOS
    - [ ] Linux (Please specify distro)
    - [ ] iOS
    - [ ] Android
 - Browser 
    - [x] Chrome
    - [ ] Edge
    - [ ] Firefox
    - [ ] Safari
 - Frontend
    - [x] Jupyter Notebook
    - [ ] Jupyter Lab
    - [ ] nteract
    - [ ] Visual Studio Code 
    - [ ] Visual Studio Code Insiders
    - [ ] Visual Studio
    - [ ] Other (please specify)

### Screenshots
![image](https://user-images.githubusercontent.com/24267968/224748382-bc0bfc5e-1cd8-4aa8-8608-7bb9fac28242.png)

**Comment by jonsequitur** [comment]
Syntax highlighting in Jupyter is not provided or affected by the .NET Interactive kernel, so I'm marking this as external. This must be related to the Jupyter install itself.

**Comment by jonsequitur** [comment]
Yes, there's usually basic syntax highlighting for C# in Jupyter Notebook and Jupyter Lab.

**Comment by jonsequitur** [comment]
No, Jupyter doesn't use .NET.

**Comment by jonsequitur** [comment]
Are you seeing this with brand new notebooks created in Jupyter?

**Comment by ibobak** [comment]
Now is 2024-03-06, I have exactly the same problem as described above.  Still not solved.

**Comment by XiaomoWu** [comment]
Interestingly, I can have R syntax highlighted correctly but couldn't do that for Python (pylance).
